pkg/controllers/hpareplicassyncer/hpa_replicas_syncer_controller.go

```diff
@@ -143,3 +150,12 @@ func (r *HPAReplicasSyncer) updateScaleIfNeed(ctx context.Context, workloadGR sc
 
 	return nil
 }
+
+func findConditionTrue(status autoscalingv2.HorizontalPodAutoscalerStatus, conditionType autoscalingv2.HorizontalPodAutoscalerConditionType) bool {
```

Comment:
How about using this function `IsStatusConditionTrue` in the `vendor/k8s.io/apimachinery/pkg/api/meta/conditions.go` file?

Reply:
No, `HorizontalPodAutoscalerCondition` is a special `condition` , so we can't use this function.

Reply:
Ok, thx~